pageobjects/MainPage.js

```diff
@@ -16,9 +16,14 @@ class MainPage extends BasePage {
       buttonName: 'Submit',
       buttonLocator: '//button[@type="submit"] | //*[class="sb-searchbox__button"]',
     },
+    CalendarButton: {
+      buttonName: 'Calendar',
+      buttonLocator: '.ed2ff9f661',
+    },
   };
 
   firstLocation = (locationName) => `//*[normalize-space()="${locationName}"]`;
```

Comment:
The name of the function should be a verb, example: getFirstLocation(), findFirstLocation(), etc.

Reply:
chooseLocation